ingestion/src/metadata/ingestion/source/database/query_parser_source.py

```diff
@@ -108,6 +108,9 @@ def get_filters(self) -> str:
             return f"{self.filters} AND {self.source_config.filterCondition}"
         return self.filters
 
+    def get_engine(self):
+        yield self.engine
```

Comment:
we would need the same logic for the Stored Procedures right? in there we are also listing the queries in the `stored_procedures_mixin`

Reply:
makes sense, we can do it in a separate pr as we will have to cherrypick this one to 1.1.5

Reply:
sure!